components/EpisodeListingOverviewDay3.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, FunctionComponent } from "react";
+import { StyleSheet } from "react-native";
+import { View, Text } from "./Themed";
+
+export const EpisodeListingOverviewDay3: FunctionComponent = () => {
+  return (
+    <View style={episodeListingStyles.container}>
+      <Text style={episodeListingStyles.title}>Great!</Text>
+      <Text style={episodeListingStyles.subheader}>Let’s start recording</Text>
+      <Text style={episodeListingStyles.text}>
+        For this first recording, we would like you to please tell us the names
+        of all of your Episodes from Day 2. {"\n\n"}If you cannot remember the
+        exact title, you can try to identify the episode by a short, few word
+        description. {"\n\n"}
+        <Text style={episodeListingStyles.doNot}>Do not</Text> tell us any
+        episodes from today, only from Day 2.
+      </Text>
+    </View>
```

Comment:
This seems to be the same (other than the day #) as the EpisodeListing for day 2. Should we combine these two?

Reply:
This is a very valid point and we have removed this file, adding a prop to the EpisodeListingOverview component.